templates/whatson/event_page.html

```diff
@@ -1,3 +1,171 @@
 {% extends 'base_page.html' %}
 
-{% load static i18n wagtailcore_tags wagtailimages_tags wagtailmetadata_tags %}
\ No newline at end of file
+{% load static i18n wagtailcore_tags wagtailimages_tags wagtailmetadata_tags %}
+
+{% block html_class %}tna-template--pink-accent{% endblock %}
+
+{# Breadcrumb is included inside the header area #}
+{% block breadcrumb %}
+{% endblock %}
+
+{% block content %}
+  {% comment %} Area for hero component {% endcomment %}
+
+  <div class="event-details-banner">
+    <div class="tna-container tna-container--align-center tna-container--space-between">
+      <div class="tna-column tna-column--width-3-4 tna-column--container">
+        <div class="event-meta event-meta--inline">
+          <ul class="event-meta__list event-meta__list">
+            {% if page.start_date %}
+              <li class="event-meta__item">
+                {% include "includes/icon.html" with name="calendar" classname="event-meta__icon" %}
+
+                <div class="event-meta__details">
+                  <p>{{ page.start_date|date:"l j F, H:i" }}</p>
+                </div>
+              </li>
+            {% endif %}
+            {% if page.venue_type %}
+              <li class="event-meta__item">
+                {% include "includes/icon.html" with name="location" classname="event-meta__icon" %}
+
+                <div class="event-meta__details">
+                  <p>{{ page.venue_type }}</p>
+
+                  {% if page.venue_address %}
+                    <address>
+                      {{ page.venue_address }}
+                    </address>
+                  {% endif %}
+                </div>
+              </li>
+            {% endif %}
+            {% if page.registration_info %}
+              <li class="event-meta__item">
+                {% include "includes/icon.html" with name="ticket" classname="event-meta__icon" %}
+
+                <div class="event-meta__details">
+                  {{ page.registration_info|richtext }}
+                </div>
+              </li>
+            {% endif %}
+          </ul>
+        </div>
+      </div>
+
+      <div class="tna-column tna-column--width-1-4">
+        <a href="/" class="tna-button tna-button--dark tna-button--icon event-details-banner__cta event-details-banner__cta--desktop">
+          Book now 
+          {% include "includes/icon.html" with name="chevron-right" classname="tna-button__icon" %}
+        </a>
+      </div>
+    </div>
+
+    <a href="/" class="tna-button tna-button--dark tna-button--icon event-details-banner__cta event-details-banner__cta--mobile">
+      Book now 
```

Comment:
When viewed without CSS, this would give us two "Book now" buttons - one after another